taglib/mpeg/id3v2/frames/ownershipframe.cpp

```diff
@@ -117,6 +117,11 @@ void OwnershipFrame::parseFields(const ByteVector &data)
 {
   int pos = 0;
 
+  // Need at least 1 byte for the encoding
+  if(data.size() - pos < 1) {
```

Comment:
`pos` is always `0` here, so I would prefer `if(data.isEmpty()) {`

Reply:
Ah, yeah. I was following the convention of other uses of `pos` below, like the `pos += 1;` that could be `pos = 1;` right after the encoding read.